src/ui/components/Card/index.js

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import React, { PropTypes } from 'react';
+
+import './Card.scss';
+
+
+export default class Card extends React.Component {
+  static propTypes = {
+    children: PropTypes.node,
+    footer: PropTypes.node,
+    header: PropTypes.node,
+  }
+
+  render() {
+    const { children, footer, header } = this.props;
+
+    return (
+      <div className="Card" ref={(ref) => { this.cardContainer = ref; }}>
+        {header ? (
+          <h2 className="Card-header" ref={(ref) => { this.header = ref; }}>
+            {header}
+          </h2>
+        ) : null}
+
+        {children ? (
+          <section className={classNames(
```

Comment:
Should this be a section? I feel a div would make sense. Perhaps `Card` should be a section?